db/seeds/dev/03_weapons.js

```diff
@@ -0,0 +1,33 @@
+const weaponsData = require('../../../data/weapons');
+
+const getClassID = async (knex, weapon) => {
+  const className = await knex('classes').where({
+    name: weapon.class
+  }).returning('id');
+  return className[0]['id'];
+}
+
+const createWeapon = async (knex, weapon) => {
+  const classID = await getClassID(knex, weapon);
+
+  return knex('weapons').insert({
+    name: weapon.name,
+    type: weapon.type,
+    damage: weapon.damage,
+    class_id: classID
+  });
+}
```

Comment:
`createWeapon` function creates weapon object with necessary data and insert it to `weapons` table.